x-pack/plugins/security_solution/cypress/data/detection_engine.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type {
+  From,
+  RiskScore,
+  Severity,
+  Threat,
+  ThreatSubtechnique,
+  ThreatTechnique,
+} from '@kbn/securitysolution-io-ts-alerting-types';
+import type {
+  Description,
+  FalsePositives,
+  Index,
+  Interval,
+  Name,
+  Note,
+  Query,
+  References,
+  Tags,
+} from '../../common/detection_engine/schemas/common';
+
+export const getQuery = (): Query => {
```

Comment:
@MadameSheema is there a special point in using parameterless functions for the default test data? As far as I see such functions can be replaced with constants, for example for `getQuery` it will be

```
export const DEFAULT_QUERY = 'host.name: *' as const;
```

The name `DEFAULT_QUERY` can vary but should represent the intent. Which also can be just `QUERY`.

Reply:
`SOME_QUERY` might make sense when you don't care about the actual value.